tools/system_libs.py

```diff
@@ -310,13 +309,12 @@ def build_objects(self, build_dir):
         cmd = [shared.EMCC]
       else:
         cmd = [shared.EMXX]
+
+      cmd += cflags
       if ext in ('.s', '.S'):
```

Comment:
I think the idea was that cflags should not be passed to assembly files.   Perhaps this goes back to when we only uses `-s` where the pre-processor is not run and do `-D` and `-I` flags are meaning less.   Assuming there are not resulting warnings through I guess its harmless.

Reply:
I did not observe any build warnings. The reason I changed this was to get the `-D` flags into the files, so those are not meaningless.

Reply:
They are meaningless for `.s` files (lowercase), but meaningful for `.S` files (uppercase).  But its not a big deal to pointlessly pass them.